packages/suite/src/views/settings/SettingsDevice/MultiShareBackup.tsx

```diff
@@ -0,0 +1,36 @@
+import { HELP_CENTER_RECOVERY_SEED_URL } from '@trezor/urls';
+
+import { openModal } from 'src/actions/suite/modalActions';
+import {
+    ActionButton,
+    ActionColumn,
+    SectionItem,
+    TextColumn,
+    Translation,
+} from 'src/components/suite';
+import { useDispatch } from 'src/hooks/suite';
+
+export const MultiShareBackup = () => {
+    const dispatch = useDispatch();
+
+    const handleClick = () => dispatch(openModal({ type: 'multi-share-backup' }));
+
+    return (
+        <SectionItem>
+            <TextColumn
+                title={<Translation id="TR_MULTI_SHARE_BACKUP" />}
+                description={<Translation id="TR_MULTI_SHARE_BACKUP_DESCRIPTION" />}
+                buttonLink={HELP_CENTER_RECOVERY_SEED_URL} // TODO: replace link
```

Comment:
Forgotten todo?

Reply:
Waiting for the correct URL, the article does not exist yet.